tests/Sulu/Component/Content/Document/Query/StructureQueryBuilderConverterTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Sulu\Component\Content\Document\Query;
+
+use Sulu\Component\Content\Document\Query\StructureQueryBuilderConverter;
+
+class StructureQueryBuilderConverterTest extends \PHPUnit_Framework_TestCase
```

Comment:
Add note explaining that the converter is primarily tested in the functional test.